### DISCUSSION TITLE
Integrating Gecode into existing application

### DISCUSSION THREAD

**Discussion by JoergStrebel** [Gecode/gecode#135] [Q&A]
Hallo!

I would like to integrate Gecode with KMines, a Minesweeper clone for the KDE Desktop environment. It is supposed to act as a solver to determine the solvability of a game state ( see https://github.com/JoergStrebel/minesweeper for details).
My target Linux distribution openSUSE does not offer Gecode as a installable package, so I cannot use the Gecode shared libraries, but I have to integrate Gecode directly with the application. I would try to combine the Gecode source code with my application code.

Are there any recommendations for the above scenario? I would like to avoid to tear apart the Gecode automake / autoconf files, but it is clear that I only need a part of the full Gecode source code.

**Comment by zayenz** [comment]
See discussion on mailing list: https://groups.google.com/g/gecode/c/UzjhYx0msYw
